### PR TITLE
add support for early terminate secotrs manually

### PR DESCRIPTION
**Pull request by ozhtdong**

### CODE REVIEW

File: cmd/lotus-storage-miner/proving.go [hunk 1]
```diff
@@ -371,3 +379,111 @@ var provingDeadlineInfoCmd = &cli.Command{
...
+	Name:  "terminate",
+	Usage: "terminate specied sector early",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "really-do-it",
+			Usage: "pass this flag if you know what you are doing",
+		},
+		&cli.StringFlag{
+			Name:  "sectors",
+			Usage: "specied sectors[xx,xx,xx]",
```
Comment (RobQuistNL) [inline cmd/lotus-storage-miner/proving.go:393]: Typo? :)

**Review by RobQuistNL — CHANGES_REQUESTED** [review]
The current way to call it is;

```
lotus-miner proving terminate --sectors="437" --really-do-it 1
```

The current other sector edit commands are as follows:
```
lotus-miner sectors remove --really-do-it 437
```

I think you might want to move the command to the `sectors` endpoint instead of `proving`, and replace the `sectors` input. This would also be the only command that allows multiple sectors to be given in a `,` seperated way - so I'd either remove that option, or add it to all other.

Then there's also the required `--sectors` flag - whereas in other we can just use `--really-do-it`

Also, `--really-do-it` shouldn't be visible in the API output, but only visible through source code (enforcing people to know a bit of programming before they use it ;) )

I also heard that it might be better to put this entire step into the FSM as well (just like the other states).

Last but not least - Thank you! Trying to terminate 1 broken sector early now :)

**Comment by RobQuistNL** [comment]
Functionally verified in 1.0.0;
https://filfox.info/en/message/bafy2bzaceacz63flwr5fhy4kafjkdiga72fpf6ujvjnnfidnypqfso5inn7tq

**Comment by arajasek** [comment]
Superseded by #4507

**Comment by jennijuju** [comment]
@ozhtdong Thank your for the PR first of all. Just want to let you know that we moved your implementation to lotus-shed with some ui changes(to make things more consistent with `lotus-miner sectors` cli as @RobQuistNL suggested). We moved it to `lotus-shed` so that this can be used as a utility cli for miners who wants to terminate the sectors on chain immediately. And we cant merge your pr yet b/c we will have to modify things in fsm.go for upgrading the sector status too.

**Comment by ozhtdong** [comment]
@RobQuistNL  all right, I agree with and thank you for your modify, you can close this pr.

**Pull request closed without merging**